### PR TITLE
fix the issue #1215

### PR DESCRIPTION
**Pull request by MahendraNG**
I have created a file to generate random address and send message from that address to a test address. Also, got the acknowledgement and synchronisation status and connection status.

### CODE REVIEW

File: src/helper_test.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+        message = 'Hello, this is the new asasf message'.encode('base64')
+
+        try:
+            ack_data = api.sendMessage(tosend, fromsend, subject, message)
+            connectedhosts = len(network.stats.connectedHostsList())
+            print "synchronisation in progress..."
+            print "Connection Status: ", connectedhosts
+            while pendingDownload() > 0:
+                print "synchronisation in progress..."
+                print "Connection Status: ", connectedhosts
```
Comment (PeterSurda) [inline src/helper_test.py]: you should get connected hosts here again

File: src/helper_test.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+        label = 'Test-' + str(time.time())
+        random_address = api.createRandomAddress(base64.encodestring(label))
+        fromsend = random_address
+        tosend = "BM-2cWyUfBdY2FbgyuCb7abFZ49JYxSzUhNFe"
+        subject = 'subject!'.encode('base64')
+        message = 'Hello, this is the new asasf message'.encode('base64')
+
+        try:
+            ack_data = api.sendMessage(tosend, fromsend, subject, message)
+            connectedhosts = len(network.stats.connectedHostsList())
```
Comment (PeterSurda) [inline src/helper_test.py]: maybe here you can wait until connectedHosts is sufficient, like 4

File: src/helper_test.py [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+class RandomNumber(object):
+    """docstring for RandomNumber."""
+
+    def __init__(self):
+        """Initialize the variables."""
+        super(RandomNumber, self).__init__()
+
+    def generateRandomNumber(self):
+        """Generate random address and send messages."""
+        api = xmlrpclib.ServerProxy("http://username:password@localhost:8442/")
```
Comment (PeterSurda) [inline src/helper_test.py]: these attributes should probably be retrieved via BMConfigParser

File: src/helper_test.py [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+        tosend = "BM-2cWyUfBdY2FbgyuCb7abFZ49JYxSzUhNFe"
+        subject = 'subject!'.encode('base64')
+        message = 'Hello, this is the new asasf message'.encode('base64')
+
+        try:
+            ack_data = api.sendMessage(tosend, fromsend, subject, message)
+            connectedhosts = len(network.stats.connectedHostsList())
+            print "synchronisation in progress..."
+            print "Connection Status: ", connectedhosts
+            while pendingDownload() > 0:
```
Comment (PeterSurda) [inline src/helper_test.py]: you should also wait until the message status turns "ackreceived"

File: src/helper_test.py [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+
+
+class RandomNumber(object):
+    """docstring for RandomNumber."""
+
+    def __init__(self):
+        """Initialize the variables."""
+        super(RandomNumber, self).__init__()
+
+    def generateRandomNumber(self):
```
[the remark this answers is not in the document]
Comment (g1itch) [reply src/helper_test.py]: random method name (:

File: src/helper_test.py [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+import network.stats
+from network.stats import pendingDownload
+
+
+class RandomNumber(object):
+    """docstring for RandomNumber."""
+
+    def __init__(self):
+        """Initialize the variables."""
+        super(RandomNumber, self).__init__()
```
[the remark this answers is not in the document]
Comment (MahendraNG) [reply src/helper_test.py]: I have removed this.

File: src/bitmessagemain.py [hunk 7]
```diff
@@ -423,7 +424,7 @@ def usage(self):
   -h, --help            show this help message and exit
   -c, --curses          use curses (text mode) interface
   -d, --daemon          run in daemon (background) mode
-  -t, --test            dryrun, make testing
+  -t, --test            this is only for developers for testing purpose
```
[the remark this answers is not in the document]
Comment (MahendraNG) [reply src/bitmessagemain.py]: I have changed this to 'this is only for developer to test the application in testing mode'

File: src/class_objectProcessor.py [hunk 8]
```diff
@@ -28,6 +28,7 @@
 import tr
 from debug import logger
 import l10n
+from addresses import calculateInventoryHash
```
[the remark this answers is not in the document]
Comment (MahendraNG) [reply src/class_objectProcessor.py]: I have removed this

**Pull request closed without merging**